Google Scholar.js

```diff
@@ -828,7 +844,11 @@ ItemFactory.prototype.getCourt = function () {
 	if (m) {
 		this.v.court = m[2].replace(/_/g, " ");
 		if (m[1]) {
-			this.v.extra = "{:jurisdiction: " + m[1] + "}";
+			if (!Zotero.Utilities.setMultiField) {
+				this.v.jurisdiction = m[1];
```

Comment:
Isn't this backwards? `jurisdiction` is MLZ-specific, not the other way around. But furthermore, doesn't MLZ store this data in `extra` anyway and then interpret it as a field (for syncing reasons)? And if no, then shouldn't MLZ be parsing out data from `extra` on import and moving it to appropriate field? I'm not terribly against this in GS (this is your baby), but I don't think we'll be introducing MLZ-specific logic in other translators, so it seems like a broader solution for MLZ would be worthwhile.